client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -233,11 +237,16 @@ class ImageEditorCanvas extends Component {
 			canvasX = -50 * widthRatio - 100 * leftRatio,
 			canvasY = -50 * heightRatio - 100 * topRatio;
 
+		const canvasOffsetTop = canvas.offsetTop;
+		const canvasOffsetLeft = canvas.offsetLeft;
+		const canvasOffsetWidth = canvas.offsetWidth;
+		const canvasOffsetHeight = canvas.offsetHeight;
+
```

Comment:
@retrofox it's a canvas node, see https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetTop